data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v5.1.xml

```diff
@@ -4,17 +4,41 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
 
     <changeSet author="R3.Corda" id="ledger-utxo-creation-v5.1">
-       <createIndex
-           indexName="utxo_visible_transaction_state_idx_consumed"
-           tableName="utxo_visible_transaction_state">
-           <column name="consumed"/>
-       </createIndex>
+        <createIndex
+                indexName="utxo_visible_transaction_state_idx_consumed"
+                tableName="utxo_visible_transaction_state">
+            <column name="consumed"/>
+        </createIndex>
```

Comment:
```suggestion
        <createIndex indexName="utxo_visible_transaction_state_idx_consumed"
                     tableName="utxo_visible_transaction_state">
            <column name="consumed"/>
        </createIndex>
```

I can see why you've changed the formatting, but the above seems consistent with the Liquibase styling convention (and what is used in other C5 Liquibase files). I'd suggest applying this to all of the entries in this file.